src/platform/packages/shared/kbn-esql-ast/src/types.ts

```diff
@@ -372,6 +373,20 @@ export interface ESQLSource extends ESQLAstBaseItem {
   selector?: ESQLStringLiteral | undefined;
 }
 
+/**
+ * Represents any expression wrapped in parentheses.
+ *
+ * ```
+ * FROM ( <query> )
+ * ```
+ */
+export interface ESQLParens extends ESQLAstBaseItem {
+  type: 'parens';
+  child: ESQLAstExpression;
+  openParenLocation: ESQLLocation;
+  closeParenLocation?: ESQLLocation;
```

Comment:
We already have `location` on `ESQLAstBaseItem`. The start of char in that range will be `(` and the last char will be `)`, or am I missing something? Why not just leave `location` as with al other nodes?